README.md

```diff
@@ -1,10 +1,64 @@
 An introductory JavaScript workshop for beginners.
 
-This workshop is a [Jekyll](https://jekyllrb.com/) site.
+## Slides
 
-It is hosted using Github Pages, so the content is in the `gh-pages` branch, not `master`.
+Check on slides, click [here](https://goo.gl/5HNvxD)
 
-To build the workshop locally:
+## Feedback
+
+To leave feedback click [here](https://docs.google.com/forms/d/e/1FAIpQLSdoOaviRIqsDPi1ZPTvTDVEzeQrSiEoDhsS0tmAIQZmkLkvxw/viewform?c=0&w=1)
+
+## How to use
+
+If you are familiar with git, you can clone this repository to your machine.
+
+If you don't know what git is, relax. You can easily download the folder on your machine - go to
+the 'releases' tab over the yellow line on the page and click the link 'Source code (zip)'.
+Unzip it (Extract) and open the folder, don't open files inside the .zip.
+
+Start from the `README.md` file, then open `js/level1.js`.
+
+To see the web-page in your browser, open `index.html` by double clicking on it, if you see
+an option 'open in browser' then choose that. Preferably use Chrome, but Firefox and Safari will work as well.
+
+Follow the instructions in `js/level1.js` and type code in your Text Editor (this is where your code lives and you can write, edit and delete code).
```

Comment:
Based on the current folder structure in this branch/PR, we'll need to update the README instructions to open the `muses-code-js-website/script.js` file.